src/web/src/pages/_app.tsx

```diff
@@ -27,11 +27,27 @@ const App = ({ Component, pageProps }: AppProps) => {
     if (jssStyles) {
       jssStyles.parentElement?.removeChild(jssStyles);
     }
-  }, []);
+
+    // Enable the stylesheet for the syntax highlighting depending on the theme
+    if (preferredTheme === 'dark') {
+      document.querySelector('#light-stylesheet')?.setAttribute('disabled', 'disabled');
```

Comment:
`.disabled = true` see https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet/disabled

Reply:
Question: should this happen within a `useEffect` @DukeManh?

Reply:
Yeah, it should be useEffect of hooks/use-preferred-theme